Tasks/PowerShellOnTargetMachines2/JobHelper.ps1

```diff
@@ -0,0 +1,33 @@
+function Run-RemoteScriptJobs {
+    [CmdletBinding()]
+    Param(
+        [System.Management.Automation.Runspaces.PSSession[]] $sessions,
+        [scriptblock] $script,
+        [System.Array] $scriptArguments,
+        [switch] $ignoreLASTEXITCODE
```

Comment:
I don't see this being used.

Reply:
yes, this is removed as part of later PR's.